tools/start-container

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
```

Comment:
'run-container' is a similar name to 'start-container', and the two scripts even look the same at first glance. This will probably confuse someone someday. 

I'd suggest:
* adding explanatory comments to the top, or
* rewriting in Python, providing docstrings